libs/k8s-client/tests/ApiClient/ConfigMapsApiClientFunctionalTest.php

```diff
@@ -0,0 +1,170 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Keboola\K8sClient\Tests\ApiClient;
+
+use Keboola\K8sClient\ApiClient\ConfigMapsApiClient;
+use Kubernetes\API\ConfigMap as ConfigMapsApi;
+use Kubernetes\Model\Io\K8s\Api\Core\V1\ConfigMap;
+use Kubernetes\Model\Io\K8s\Apimachinery\Pkg\Apis\Meta\V1\DeleteOptions;
+use Kubernetes\Model\Io\K8s\Apimachinery\Pkg\Apis\Meta\V1\Status;
+use PHPUnit\Framework\TestCase;
+use RuntimeException;
+
+class ConfigMapsApiClientFunctionalTest extends TestCase
+{
+    /**
+     * @template-use BaseNamespaceApiClientTestCase<ConfigMapsApi, ConfigMapsApiClient>
+     */
+    use BaseNamespaceApiClientTestCase;
+
+    public function setUp(): void
+    {
+        //@TODO resolve problems with root crt that is also saved in configmap in same namespace
+        parent::setUp();
+        $this->setUpBaseNamespaceApiClientTest(
+            ConfigMapsApi::class,
+            ConfigMapsApiClient::class,
+        );
+    }
+
+    private function cleanupK8sResources(float $timeout = 30.0): void
```

Comment:
Tohle má speciální logiku, protože k8s automaticky do každýho namespace hodí `kube-root-ca.crt` a ten smazat nejde.
Podobnou logiku bysme mohli udělat v `SecretsApiClientFunctionalTest`. Pak bysme mohli mít všechno v jednom namespace a nemít ten `k8s_client_account` namespace.

Reply:
Nakonec jsem to trochu upravil a přesunul do https://github.com/keboola/platform-libraries/pull/95/files#diff-bb9b2be1e2f958c7fd75ec8a2c1416b3e153eb769e4987b3c59bab9a8c68cb71, protože v Secretech je taky uloženej service account kterej nechceme mazat.